openpgp/packet/signature.go

```diff
@@ -927,8 +929,17 @@ func (sig *Signature) Sign(h hash.Hash, priv *PrivateKey, config *Config) (err e
 			sig.DSASigS = new(encoding.MPI).SetBig(s)
 		}
 	case PubKeyAlgoECDSA:
-		sk := priv.PrivateKey.(*ecdsa.PrivateKey)
-		r, s, err := ecdsa.Sign(config.Random(), sk, digest)
+		var r, s *big.Int
+		var err error
```

Comment:
I think this is incorrect as it shadows the return parameter, hiding errors returned below
```suggestion
```

Reply:
Fixed.